packages/jaeger-ui/src/components/TracePage/TraceGraph/TraceGraph.css

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright (c) 2017 The Jaeger Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.TraceGraph--experimental {
+  background-color: #a00;
+  position: absolute;
+  top: 122px;
+}
+.TraceGraph--experimental a {
+  color: #fff;
+  padding: 10px 15px;
+}
+
+.TraceGraph--graphWrapper {
+  bottom: 0;
+  cursor: move;
+  left: 0;
+  overflow: auto;
+  position: absolute;
+  right: 0;
+  top: 0;
+  display: flex;
+}
+
+.TraceGraph--sidebar-container {
+  display: flex;
+  z-index: 1;
+}
+
+.TraceGraph--menu {
+  cursor: pointer;
+  padding-right: 1rem;
+  padding-top: 1rem;
+}
+
+.TraceGraph--menu > li {
+  list-style-type: none;
+  text-align: right;
+  padding-bottom: 0.3rem;
+}
+
+.TraceGraph--sidebar {
+  cursor: default;
+  box-shadow: -1px 0 #00000030;
```

Comment:
[20% of browsers](https://caniuse.com/#feat=css-rrggbbaa) don't support `#rrggbbaa` color format.